src/common/common.js

```diff
@@ -196,9 +196,15 @@ const Localizer = (function () {
         // localize main content
         if (tag !== "") {
             const translatedMessage = browser.i18n.getMessage(getMessageTag(tag));
+            const isHTML = translatedMessage.startsWith("!HTML!");
             // only set message if it could be retrieved, i.e. do not override HTML fallback
             if (translatedMessage !== "") {
-                elem.innerHTML = translatedMessage;
+                if (isHTML) {
+                    const normalizedMessage = translatedMessage.replace(/!HTML!/, "").trim();
```

Comment:
Hmm, I am not sure whether trimming the whole string is always want you want. I mean, in the current code base there is no real need for deliberate spaces at the end or start of a string AFAIK, but maybe there will be in the future.

Although not standardized maybe [trimLeft](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimLeft) is all right here, because it seems to have widespread support anyway. (Or could it trim only one space after the string, if needed…?)

Reply:
Alright. I was not aware of this method.